spec/definitions.yaml

```diff
@@ -318,6 +318,37 @@ FeatureFlagStatuses:
       type: array
       items:
         $ref: '#/definitions/FeatureFlagStatus'
+FeatureFlagStatusForQueriedEnvironment:
+  type: object
+  properties:
+    name:
```

Comment:
The only difference between `FeatureFlagStatus` and `FeatureFlagStatusForQueriedEnvironment` is `FeatureFlagStatusForQueriedEnvironment` does not have _links. Is there a better way to dry this up?